rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/DefenceResponseJsonMapper.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.party.Individual;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.rpa.DateFormatter;
+import uk.gov.hmcts.cmc.rpa.mapper.json.NullAwareJsonObjectBuilder;
+
+import javax.json.JsonObject;
+
+import static uk.gov.hmcts.cmc.rpa.mapper.helper.Extractor.extractFromSubclass;
+
+@Component
+public class DefenceResponseJsonMapper {
+
+    @Autowired
+    private final DefendantJsonMapper mapDefendant;
+
+    public DefenceResponseJsonMapper(DefendantJsonMapper mapDefendant) {
```

Comment:
Please remove constructor, auto wiring fields should be enough.

Please follow the same pattern in other places.